src/dynamics/mpas/driver/dyn_mpas_subdriver.F90

```diff
@@ -10,7 +10,12 @@ module dyn_mpas_subdriver
     use, intrinsic :: iso_fortran_env, only: output_unit
 
     ! Modules from external libraries.
+#ifdef MPAS_USE_MPI_F08
+    use mpi_f08, only: mpi_comm_null, mpi_comm_rank, mpi_success, &
+                       mpi_comm_type => mpi_comm, operator(==), operator(/=)
```

Comment:
Is the `/=` operator needed?

Reply:
No, it is not needed. I have removed it in https://github.com/ESCOMP/CAM-SIMA/pull/288/commits/7e059d1f16b498035095a519f1dd244c19032049.